docs/assets/js/search.js

```diff
@@ -9,43 +9,56 @@ const searchResultsElm = document.getElementById( 'search-results' );
  * Update page markup with search results.
  * @param {Array} results - A list of search result hits as objects.
  * @param {Object} store - search index/meta data store in the window object.
+ * @returns {boolean} True if there are search results, false otherwise.
  */
 function displaySearchResults( results, store ) {
   // Are there any results?
-  if ( results.length ) {
-    let appendString = '';
-
-    // Iterate over the results.
-    for ( let i = 0; i < results.length; i++ ) {
-      const item = store[results[i].ref];
-      appendString += '<li><a href="../' + item.url + '"><h3>' + item.title + '</h3></a>';
-
-      // Show some preview text under each search results item.
-      let previewText = '';
-      const searchMatchWordFragment = Object.keys( results[i].matchData.metadata )[0];
-      const searchMatchFields = results[i].matchData.metadata[searchMatchWordFragment];
-
-      // Remove fields that should never appear as the preview.
-      delete searchMatchFields.id;
-      delete searchMatchFields.title;
+  if ( results.length === 0 ) {
+    displayNoSearchResults( searchResultsElm, results.searchTerm );
+    return false;
+  }
 
-      previewText = item[Object.keys( searchMatchFields )[0]];
+  // Search results content will be placed in here.
+  let resultsString = `
+    <p>
+      ${ results.length } result${ results.length > 1 ? 's' : '' }
+      for search of '${ results.searchTerm }'.
+    </p>
+  `;
+
+  // Iterate over the results.
+  for ( let i = 0; i < results.length; i++ ) {
+    const item = store[results[i].ref];
+    resultsString += '<li><a href="../' + item.url + '"><h3>' + item.title + '</h3></a>';
```

Comment:
If you're using a template literal above on L22 maybe use it every time you do string interpolation for consistency's sake.

Reply:
Updated in https://github.com/cfpb/design-system/pull/479/commits/2f47506c0040baa2cf8c924d7b750300f8ee560e